README.md

```diff
@@ -125,7 +135,34 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+# 1. Create a lamp with 3 wishes remaining and a genie named 'Genie'
+
+lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name: 'Genie')
+
+# 2. Create a relationship between 'Genie' and the lamp.
+
+class Lamp < ActiveRecord::Base
+  has_one: genie
+end
+
+class Genie < ActiveRecord::Base
+  belongs_to: lamp
```

Comment:
Close. `belongs_to` does not take a colon, but `lamp` should. So the line will read something like...

```rb
belongs_to :lamp
```